modules/uhttp/filters.go

```diff
@@ -80,6 +81,22 @@ func tracingServerFilter(host service.Host) FilterFunc {
 	}
 }
 
+// authFilter authorizes services based on configuration
+func authFilter(host service.Host) FilterFunc {
+	return func(ctx context.Context, w http.ResponseWriter, r *http.Request, next Handler) {
+		fxctx := &fxcontext.Context{
+			Context: ctx,
+		}
+		authClient := uauth.Client()
+		err := authClient.Authorize(fxctx)
+		if err != nil {
+			host.Metrics().Counter("http.authorize.fail").Inc(1)
+			fxctx.Logger().Error(uauth.ErrAuthorization, "error", err)
```

Comment:
Should this `return` and / or write some status code rather than continuing on to `next` if auth fails?